cmd/server/server.go

```diff
@@ -51,13 +51,26 @@ func NewCommand() *cobra.Command {
 }
 
 func serverFunc(cmd *cobra.Command, args []string) (err error) {
-	lcfg := logutil.GetDefaultZapLoggerConfig()
-	lcfg.Level = zap.NewAtomicLevelAt(logutil.ConvertToZapLevel(logLevel))
-	logger, err := lcfg.Build()
+	if logDir == "" {
```

Comment:
where is logDir specified?

Reply:
probably there should be a flag for it. also for control.

Reply:
Not for control, as that is a single independent run for every command, and it could quickly become messy I think.